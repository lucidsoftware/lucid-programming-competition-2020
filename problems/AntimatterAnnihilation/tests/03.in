matter_insantimatteride_antiantimatterantimattermatter_anantimattertiantimattermatantimatterter_other_matter