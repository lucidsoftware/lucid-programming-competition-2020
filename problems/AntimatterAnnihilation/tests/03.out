matter_inside___other_matter